### PR TITLE
Untyped attachments

### PR DESCRIPTION
**Pull request by Frozen**

### CODE REVIEW

File: cmd/node/node.go [hunk 1]
```diff
@@ -319,6 +319,8 @@ func main() {
 	n := node.NewNode(services, declAddr, bindAddr, proto.NewTimestampFromUSeconds(outdatePeriod))
 	go n.Run(ctx, parent, services.InternalChannel)
 
+	go sched.Reschedule()
```
Comment (alexeykiselev) [inline cmd/node/node.go:322]: What is the point of doing reschedule here? There is no connection and no consensus at this moment. I think this is pointless.

File: pkg/ride/evaluator/reader/reader.go [hunk 2]
```diff
@@ -148,6 +148,10 @@ func (a *BytesReader) ReadBytes() []byte {
 	return out
 }
 
+func (a *BytesReader) Content() []byte {
```
Comment (alexeykiselev) [inline pkg/ride/evaluator/reader/reader.go:151]: No usages

File: cmd/node/node.go (hunk 1 above)
Comment (Frozen) [reply cmd/node/node.go:322]: Node can mining without connections. There should be initial schedule request.

File: pkg/ride/evaluator/reader/reader.go (hunk 2 above)
Comment (Frozen) [reply pkg/ride/evaluator/reader/reader.go:151]: Its used by foreign project.

**Pull request merged**